File: 251011/숫자 피라미드 2/number-pyramid-2.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int a;
    cin >> a;
    int prt = 1;

    for(int i = 1; i <= a; i++) {
        for(int j = 1; j <= i; j++) {
            cout << prt << " ";
            prt++;
        }
        cout << "\n";
    }
    return 0;
}

File: 251113/숫자 피라미드 2/number-pyramid-2.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        // Please write your code here.
        int a = int.Parse(Console.ReadLine());
        int count = 1;
        for(int i = 1; i <= a; i++) {
            for(int j = 0; j < i; j++) {
                Console.Write(count + " ");
                count++;
            }
            Console.WriteLine("");
        }
    }
}
